src/utils/env.ts

```diff
@@ -13,3 +13,7 @@ export function isStagingEnv(): boolean {
 export function isProductionEnv(): boolean {
   return process.env.NODE_ENV === 'production' && !isStagingEnv()
 }
+
+export function isSentryEnabled(): boolean {
+  return process.env.REACT_APP_SENTRY_DISABLED !== 'true'
```

Comment:
Have you verified that this is `'true'` and not `true`?

Reply:
Yes, `dotenv` converts everything to a string, including numbers actually